math/mathcore/inc/Math/GaussIntegrator.h

```diff
@@ -43,7 +43,7 @@ class GaussIntegrator: public VirtualIntegratorOneDim {
 public:
 
    /** Destructor */
-   ~GaussIntegrator() override;
+   virtual ~GaussIntegrator() override;
```

Comment:
The override implies that the function in the base class is already set to be virtual and thus it is redundant, isn't it?

Reply:
Exactly, so the memory leak must be elsewhere. If you think the memory leak is because of the new ROOT version, and you need help identifying the leak, please open a GitHub issue with some more context and in the best case a reproducer, so we can fix it!

Reply:
See also:
https://stackoverflow.com/questions/43466863/isnt-virtual-keyword-redundant-when-override-or-final-specifiers-are-used